js/src/forum/utils/popupUtils.ts

```diff
@@ -0,0 +1,26 @@
+import type ForumApplication from 'flarum/forum/ForumApplication';
+
+export function openOAuthPopup(app: ForumApplication, attrs: Record<string, any>) {
+  const fullscreen = app.forum.attribute('fof-oauth.fullscreenPopup');
+  if (fullscreen) {
+    window.open(app.forum.attribute('baseUrl') + attrs.path, 'logInPopup', 'fullscreen=yes');
```

Comment:
As we are making this more extensible, I think it would be useful to be able to control the `fullscreen` param here. We should still make it `yes` by default, but let's turn this into a property we can pass into `openOAuthPopup`

Reply:
Yeah so we can control the fullscreen param in theory by changing the setting in the admin panel. Depending if thats enabled or not it actually executes totally different logic.

Reply:
I guess the question would be if I should clean this up a little and remove the need for an if else statement (depending on fullscreen setting)?

Reply:
🤦‍♂️ sorry, was rushing thru this :(

Yeah, valid comment, I think it would be a nice to have for the future, but not neccessary right now (Unless you have spare time!)